securedrop-admin

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python2
+"""
+SecureDrop Admin Toolkit.
+
+For use by administrators to install, maintain, and manage their SD
+instances.
+"""
+
+import argparse
+import subprocess
+import os
+import sys
+import getpass
+
+
+SD_DIR = os.path.dirname(os.path.realpath(__file__))
+ANSIBLE_PATH = os.path.join(SD_DIR, "./install_files/ansible-base")
+SITE_CONFIG = os.path.join(ANSIBLE_PATH, "group_vars/all/site-specific")
+VENV_DIR = os.path.join(SD_DIR, ".venv")
+VENV_ACTIVATION = os.path.join(VENV_DIR, 'bin/activate_this.py')
+
+
+def error(msg, level='warning', exit=True):
+    """ Stupid wrapper function to normalize warnings """
+    print("{} - {}".format(level.upper(), msg))
+    if exit:
+        sys.exit(1)
+
+
+def activate_venv():
+    """Use to activate the local virtualenv"""
+    try:
+        execfile(VENV_ACTIVATION, dict(__file__=VENV_ACTIVATION))
+    except IOError:
+        error("Pre-requisites not in place, re-run command with 'setup'",
+              level='error')
+
+
+def sdconfig(args):
+    """Configure SD site settings"""
+    activate_venv()
+    subprocess.call(['./configure'], cwd=ANSIBLE_PATH, env=os.environ.copy())
+
+
+def envsetup(args):
+    """Install SD pre-reqs"""
+    # virtualenv doesnt exist? Install dependencies and create
+    if not os.path.exists(VENV_ACTIVATION):
+        subprocess.call(['sudo', 'su', '-c',
+                        "apt-get update && \
+                        apt-get install -y \
+                        python-virtualenv \
+                        python-pip \
+                        ccontrol \
+                        virtualenv \
+                        libffi-dev \
+                        libssl-dev \
+                        libpython2.7-dev"])
+
+        # Technically you can create a virtualenv from within python
+        # but pip can only be run over tor on tails, and debugging that
+        # along with instaling a third-party dependency is not worth
+        # the effort here.
+        subprocess.call(['torify', 'virtualenv', VENV_DIR])
```

Comment:
@fowlslegs is right; since you aren't checking the return value of `subprocess.call`, it's possible that this call could fail and your script would continue executing, which probably isn't what you want. At least `check_call` raises an exception (thereby preventing the script from continuing) if the call fails.

It would be even better to have some kind of graceful/intelligent error handling throughout this script, but that can certainly wait for follow-up issues/future releases.